3party/opening_hours/opening_hours.hpp

```diff
@@ -133,6 +134,7 @@ class Time
   void SetEvent(TimeEvent const & event);
 
   HourMinutes const & GetHourMinutes() const;
+  HourMinutes & GetHourMinutes();
```

Comment:
Do you really need a non-const version of GetHourMinutes()? If somebody will decide to change the value of HM, it will use SetHourMinutes(), right?